### PR TITLE
PXB-2515 Percona XtraBackup 8.0.25-17 release notes (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   conf.py
	modified:   release-notes.rst
	new file:   release-notes/8.0/8.0.25-17.0.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2442` : Backup cannot be decompressed using the apparmor profile
+
+* :jirabug:`PXB-2444` : Xbcloud fails to upload backup with enforcing SELinux mode
+
+* :jirabug:`PXB-2505` : Keyring is not generated properly by generate-new-master-key for component
+
+* :jirabug:`PXB-2101` : PXB 8.0 —prepare crash if exchange partition happened
```
Comment (george-lorch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: I believe Rahul and/or Manish just 'unfixed' this one

**Review by george-lorch — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+costs while providing unique features for MySQL backups.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2442` : Backup cannot be decompressed using the apparmor profile
+
+* :jirabug:`PXB-2444` : Xbcloud fails to upload backup with enforcing SELinux mode
+
+* :jirabug:`PXB-2505` : Keyring is not generated properly by generate-new-master-key for component
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: Remove this one, this is a post fix after the merge

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* :jirabug:`PXB-2445` : Initializing the libgcrypt in xbcloud
+
+* :jirabug:`PXB-2455` : xtrabackup prepare fails if checkpoint Isn is more than the last Isn
+
+* :jirabug:`PXB-2473` : SELinux errors in audit logs
+
+* :jirabug:`PXB-2496` : PXB8.0.25 backup prepare always uses the daemon_keyring_proxy_plugin without encryption
+
+* :jirabug:`PXB-2504` : Stats functionality does not work if keyring component is needed on a mysql datadir
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: Same here, we can omit those as they are post fixes from the merge

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
two comments to be fixed

**Review by mchawla16 — APPROVED** [review]
Remove PXB-2101 as it has been reopened. Add PXB-2457 and PXB-2106.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: Removed.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: removed.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst (hunk 3 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: Removed.

**Comment by patrickbirch** [comment]
2457 and 2106 added.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst (hunk 3 above)
Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst]: @patrickbirch please remove :jirabug:`PXB-2496` too

**Comment by patrickbirch** [comment]
Removed PXB-2496

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Review by mchawla16 — APPROVED** [review]

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Pull request merged**